google_bigquery_writer/writer.py

```diff
@@ -102,8 +104,14 @@ def prepare_table(
     def write_table(self, csv_file_path: str, dataset_name: str, table_definition: dict, incremental: bool = False) \
             -> List[bigquery.LoadJob]:
 
+        chunk_size = table_definition.get('chunkSize')
+        if not chunk_size:
+            chunk_size = self.DEFAULT_CHUNK_SIZE_MB
+            logging.info('Chunk size not specified, using default value: %s' % self.DEFAULT_CHUNK_SIZE_MB)
+
         if dataset_name == '' or dataset_name is None:
```

Comment:
if not dataset_name:
    raise UserException('Dataset name not specified.')

Reply:
JJ, bylo mi to jasné, ale nechtěl jsem do toho sahat víc než bylo potřeba. Chceš to předělat?

Reply:
nn, nesahejme na to. Jen mě zaujalo to, jak je to použitý i tady takhle složitě :)